testing/runner_test.ts

```diff
@@ -1,30 +1,77 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 import { test } from "./mod.ts";
+import { findTestModules } from "./runner.ts";
+import { SEP, isWindows } from "../fs/path/constants.ts";
 import { assertEquals } from "../testing/asserts.ts";
-import { getMatchingUrls } from "./runner.ts";
-import { join } from "../fs/path/mod.ts";
 
-/**
- * IMPORTANT: This file assumes it is run from root of repository.
- */
-const cwd = Deno.cwd();
-const TEST_ROOT_PATH = join(cwd, "fmt");
+const TEST_DATA_URL = new URL("testdata", import.meta.url);
+const TEST_DATA_PATH = TEST_DATA_URL.pathname
+  .slice(isWindows ? 1 : 0)
+  .replace(/\//g, SEP);
```

Comment:
What's happening here? This looks brittle. Ideally this operation be abstracted into a function with a comment.

Reply:
The `slice()` is because `new URL('file:///C:/a').pathname` is `/C:/a`, the leading slash is bad. The backslashes are necessary I think because of `globrex()`'s implementation... but they shouldn't be. I'll tinker.

Reply:
👍